internal/bgp/bgp.go

```diff
@@ -78,4 +78,5 @@ type SessionParameters struct {
 type SessionManager interface {
 	NewSession(logger log.Logger, args SessionParameters) (Session, error)
 	SyncBFDProfiles(profiles map[string]*config.BFDProfile) error
+	SyncExtraInfo(extras string) error
```

Comment:
nit: why not ExtraConfig?

Reply:
because this is implementation independent, so in the frr case we use it for adding a piece of configuration, but it might not always be the case. I wanted to keep the name a bit higher level